chemie/committees/urls.py

```diff
@@ -6,8 +6,24 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='list_all'),
-    url(r'^user-autocomplete/',views.UserAutocomplete.as_view(), name='user-autocomplete',),
-    url(r'^(?P<slug>[\w-]+)/$', views.view_committee, name='committee_detail'),
-    url(r'^(?P<slug>[\w-]+)/edit/', views.edit_description, name='edit_description'),
-    url(r'^(?P<slug>[\w-]+)/edit-members/', views.edit_committee_memberships, name='edit_memberships'),
+    url(
```

Comment:
Note to self: 
Legg til [black](https://github.com/ambv/black) så blir det lettere i framtida så vi slippe å vase gjennom "style changes".

Reply:
Smart. Også kompatibelt med VS Code